core/store/migrations/0022_create_log_config_table.go

```diff
@@ -0,0 +1,70 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+const up22 = `
+		CREATE TYPE service AS ENUM (
+			'directrequest',
+			'fluxmonitor',
+			'gasupdater',
+			'job',
+			'keeper',
+			'log',
+			'ocr',
+			'backup',
+			'pipeline',
+			'postgres',
+			'signatures',
+			'synchronization',
+			'telemetry',
+			'vrf',
+			'head_broadcaster',
+			'head_tracker',
+			'job_subscriber',
+			'prom_reporter',
+			'reaper',
+			'run_executor',
+			'run_manager',
+			'run_queue',
+			'scheduler',
+			'subscription',
+			'validators'
```

Comment:
I really wouldn't bother making this an enum. Just leave it as `text`

Reply:
roger, was thinking that this would define the services so naming doesn't get messed up and possibly used elsewhere but if no good case for it then I can remove

Reply:
We could handle any incorrect service names when setting the logger so probably redundant here anyway, only case I would see to use it for would be if we wanted the service names in the db but might be overkill~